blocks/api/registration.js

```diff
@@ -14,7 +14,7 @@ import { applyFilters } from '@wordpress/hooks';
  * Internal dependencies
  */
 import { getCategories } from './categories';
-import { validatePluginId } from '../../utils/plugins';
+import { validateNamespacedId } from '../../utils/plugins';
```

Comment:
it needs to be imported as `@wordpress/utils` and the method should be exposed in the root file of the folder.